wakuv2/waku.go

```diff
@@ -455,6 +461,32 @@ func (w *Waku) dnsDiscover(ctx context.Context, enrtreeAddress string, apply fnA
 	return nil
 }
 
+func (w *Waku) retryDnsDiscoveryWithBackoff(ctx context.Context, addr string, successChan chan<- struct{}) {
```

Comment:
I'm not so sure, but it seems that in status-go it's preferred to use `backoff.ExponentialBackOff` from `github.com/cenkalti/backoff/v3` rather than implementing it manually.

Reply:
Seems to be used only in tests. I'll do a refactoring of how backoffs are used in waku.go in a separate PR since i saw another instance of manual backoff period being done there